src/controllers/users.go

```diff
@@ -106,3 +107,12 @@ func (c *UsersController) VerifyEmailAndPassword(email string, password string)
 	}
 	return user, nil
 }
+
+func (c *UrlsController) GetUrlsByUserId(userId uint64) ([]models.Url, error) {
```

Comment:
getting URLs should not be happening in Users controller, it should happen in Urls Controller only